uv.lock

```diff
@@ -1,10 +1,18 @@
 version = 1
 requires-python = ">=3.10"
 resolution-markers = [
-    "python_full_version < '3.11'",
-    "python_full_version == '3.11.*'",
-    "python_full_version == '3.12.*'",
-    "python_full_version >= '3.13'",
+    "python_full_version < '3.11' and platform_system == 'Darwin'",
```

Comment:
Do we need all those changes in this file?

Reply:
`uv.lock` is generated automatically by `uv`, it's hard (or impossible) to control it by hand. It seems that some nvidia-related packages introduced some platform-specific versioning